provision-contest/ansible/EOC/README.md

```diff
@@ -0,0 +1,79 @@
+## DOMjudge EOC ansible helper
+This ansible playbook should automate must of the tasks DOMjudge needs to perform for the End Of Contest procedures during the ICPC WF's. It was created for the WF 2021 in Dhaka, which took place in 2022.
+
+Two types of authentication are used and supported. Session based auth and basic auth.
+ - Session based auth retrieves a CSRF token and fakes a login action.
+ - Basic auth headers are constructed and used. This is to fully mitigate [annoying basic-auth issues](https://askubuntu.com/questions/1070838/why-wget-does-not-use-username-and-password-in-url-first-time/1070847#1070847), even though the `force_basic_auth` option should already prevent these issues from arising.
+
+The full playbook can be run using the following command. See the setup section to see what the contents of facts.yml are supposed to be.
+
+```ansible-playbook main.yml --extra-vars "@facts.yml" --tags all```
+
+### Steps automated.
+The following steps from the EOC 2022-Oct document have been automated:
+
+29. `Primary pushes results.tsv into git repo`. Tag for just this action: `results.tsv`. Note, a manual commit is still needed!
+32. `Compare scoreboard.json between primary and shadow`. Tag for just this action: `scoreboard`. Note, only if other (shadow or primary) credentials are provided.
+33. `Compare awards.json between primary and CDS`. Tag for just this action: `awards`. Note, only if CDS credentials are provided.
+36. `Verify that Primary and ... System Test`. Tag for just this action: `fetch`. Note, a manual commit is still needed and the `final_standings.html` is actually the static-scoreboard zip.
```

Comment:
DOMjudge has an actual final_standings.html which is probably expected to be used here?

Reply:
Fixed